### ISSUE TITLE
darktable 3.6.1 | Cannot create HDR photo from Fuji X-S10 RAW images

### ISSUE THREAD

**Issue by cbochs** [darktable-org/darktable#10011]
**Introduction**

As the title states, I am unable to create an HDR image from a set of 7 RAW (`.RAF`) exposure-bracketed images. With the files selected, I clicked "create HDR" in Lightroom and was briefly shown a popup with the text "wrote merged HDR hdr.dng". However, no file can be found in the directory of the RAW images.

**Did you buy darktable from an application store ?**

No.

**Describe the bug/issue**

I have uploaded the logs from `darktable-log.txt` after the point of clicking "create HDR" while running with the `-d all` flag.

* Full logs: [darktable-log.txt](https://github.com/darktable-org/darktable/files/7174081/darktable-log.txt)
* Snippet after clicking "create HDR": [darktable-log-snippet.txt](https://github.com/darktable-org/darktable/files/7174056/darktable-log-snippet.txt)

At a glance I don't see much that immediately stands out to me. However, I do see the following general error logs emitted when clicking "create HDR":

```
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[dt_imageio_export_with_flags] mipmap allocation for `unused' failed
[exiv2 dt_exif_read_blob] : : Failed to open the data source: Invalid argument (errno = 22)
```

**To Reproduce**

1. Add a folder with a set of RAW images to the library
2. Select the RAW images
3. Click "create HDR"
4. Pop-up appears: "wrote merged HDR hdr.dng"
5. No merged DNG file is created

**Expected behavior**

An DNG file with the name `${FILE_NAME}-hdr.dng` should be created in the open directory.

**Platform**
_Please fill as much information as possible in the list given below. Please state "unknown" where you do not know the answer and remove any sections that are not applicable _

* darktable version : 3.6.1
* OS                : Windows 10, 64-bit
* Memory            : 32GB
* Graphics card     : GTX 1060 6GB
* Graphics driver   : 
* OpenCL installed  : 
* OpenCL activated  : 
* Xorg              : 
* Desktop           : 
* GTK+              : 
* gcc               : 
* cflags            : 
* CMAKE_BUILD_TYPE  : 

**Additional context**

 - Can you reproduce with another darktable version(s)? **Not tested**
 - Can you reproduce with a RAW or Jpeg or both? **I only have RAF files available to me, and HDR does not work with JPEG**
 - Are the steps above reproducible with a fresh edit (i.e. after discarding history)? **All images were freshly imported**
 - If the issue is with the output image, attach an XMP file if (you'll have to change the extension to `.txt`)
 - Is the issue still present using an empty/new config-dir (e.g. start darktable with --configdir "/tmp")? **Not tested**
 - Do you use lua scripts? **No**

**Comment by cbochs** [comment]
Bump?

**Comment by johnny-bit** [comment]
Do NOT bump. For bumping we have bot that marks issues that haven't received attention. And for attention - remember that darktable is made by volounteers contributing their own time to the project in manner they see fit. To make it as straightforward as possible for contributors you ought to do as much testing and provide as much info as possible. As I see you haven't updated "aditional context" part, probably haven't tried with different raw formats etc. Please do at least that.

**Comment by cbochs** [comment]
Sorry, I am new to this repo. I've updated the "additional context" section. I could hack together a test for a previous version of darktable 3.4.0, but it should be noted that the X-S10 was not supported until darktable 3.6.0.

Asides from that, I had done a bit of preliminary digging before filing the issue; unfortunately my familiarity with this project is only at the surface level so I was really only able to note the error messages I observed. If there is a better debug flag I could use (asides from `-d all`), please let me know and I can try it out.

**Comment by cbochs** [comment]
Follow-up to this issue:

When attempting to edit any RAF (from the same Fuji X-S10 camera) in lighttable, I get the following error:

```
An unhandled exception occured.
backtrace will be written to:
C:\Users\...\Temp\darktable_bt_HFE890.txt after you
click on the Ok button.
```

This is the stacktrace that was written: [darktable_bt_HFE890.txt](https://github.com/darktable-org/darktable/files/7199231/darktable_bt_HFE890.txt). Right at the top of the log:

```
darktable.exe caused an Access Violation at location 00007FFD2C896EB4 in module libgobject-2.0-0.dll Reading from location FFFFFFFFFFFFFFFF.
```

And also the UI message (before the program crashes):

![Error](https://user-images.githubusercontent.com/2467016/134091257-a8fff68d-0c2f-4d21-9144-b69be691fe53.PNG)

I might be mistaken, but I raised https://github.com/darktable-org/darktable/issues/8515 in earlier this year when using 3.4.1.1 to add camera support for the X-S10. It looks like support was added before the issue was closed, but according to https://www.darktable.org/resources/camera-support/, this camera is not supported yet? Just opening the `cameras.xml` locally in my 3.6.1 installation, there looks to be X-S10 data. 

Would anyone be able to confirm for me if the X-S10 is supported in darktable 3.6.1?

Thanks!
